README.md

```diff
@@ -1,2 +1,8 @@
 # iterativereview
 This repo is to test out Github's iterative review experience. Yay!
+
+This is a line I added.
```

Comment:
(Leaving a comment on a filtered diff view) (Line retained)

I'm glad you kept this line.

Reply:
Sure - I'll do that now